doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch

```diff
@@ -1,7 +1,5 @@
 <launch>
 
-  <include file="$(find pr2_moveit_config)/launch/demo.launch"/>
-
```

Comment:
We should keep this consistant across the tutorials.
Some of the other launch files also include `demo.launch`.